### ISSUE TITLE
Working again but are these errors normal?

### ISSUE THREAD

**Issue by heffman92** [finndev/PokeBuddy#435]
"End of stack trace from previous location" ... - In red and keeps popping up. Seems to work itself out as it continues to walk and catch pokemon and farm pokestops. But sometimes it takes a minute as the error keeps popping up. Also it isn't showing my level or inventory or XP. And it isn't recycling my items or evolving Pokemon/Transferring them. I'm guessing this is normal and will be worked out but I just wanted to check to see if others are getting this error. Thanks

Also wanted to add that things are popping up double. It's spinning my pokestops twice and showing walking location twice. It's not a serious issue but it says 0 XP as because the pokestop is spun twice so it's in cooldown period. Is anyone having this issue as well?

**Comment by WilliamcaFox** [comment]
If you start and stop the bot again in quick succession sometimes this happens with the multiple actions. Try stopping, closing and letting it rest for a minute then reopening and continuing.

**Comment by heffman92** [comment]
I tried that and now I can't even get in. Just says updating. Damn...I'll keep trying

**Comment by WilliamcaFox** [comment]
Try a fresh download, and make sure your coordinates aren't too far from the origin (in fact a  good idea is to keep your favorite farming location in a notepad somewhere)

**Comment by heffman92** [comment]
Yeah I have a set location I use. And I restarted my computer gonna see if that works. Otherwise I'll uninstall and reinstall. Are u having your level and total xp show up along with pokeballs? The only thing for me is stardust. All the other stuff is whatever it's not a big deal since the app is working, just am curious

**Comment by heffman92** [comment]
Everything's working after I restarted my computer! Yay :)
